seatunnel-connectors-v2/connector-doris/src/main/java/org/apache/seatunnel/connectors/seatunnel/doris/serialize/DorisCsvSerializer.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.seatunnel.connectors.seatunnel.doris.serialize;
```

Comment:
Why not use text-format and json-format?

Reply:
Doris sink format only supports json and csv.

Reply:
But it's also can use the ability of format modules.

Reply:
cc @EricJoy2048 @hailin0 @ashulin @Hisoka-X

Reply:
Thanks for your suggestion, I will use the format module.
At present, this is a draft and may need further improvement.